controller_manager:
  ros__parameters:
    update_rate: 1000
    use_sim_time: true

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    # milltap_controller:
    #   type: joint_trajectory_controller/JointTrajectoryController
    
    joints2_controllers:
      type: joint_trajectory_controller/JointTrajectoryController

    joints3_controllers:
      type: joint_trajectory_controller/JointTrajectoryController

    joints4_controllers:
      type: joint_trajectory_controller/JointTrajectoryController

    joints5_controllers:
      type: joint_trajectory_controller/JointTrajectoryController

    joints6_controllers:
      type: joint_trajectory_controller/JointTrajectoryController

# milltap_controller:
#   ros__parameters:
#     command_interfaces:
#       - position  
#     state_interfaces:
#       - position
#       - velocity
#     joints:
#       - joint2
#       - joint3
#       - joint4
#       - joint5
#       - joint6

joint_state_broadcaster:
  ros__parameters:
    joints:
      - joint2
      - joint3
      - joint4
      - joint5
      - joint6
                      
joints2_controllers:
  ros__parameters:
    joints:
      - joint2
    interface_name: position
    command_interfaces:
      - position  
    state_interfaces:
      - position
      - velocity

joints3_controllers:
  ros__parameters:
    joints:
      - joint3
    interface_name: position
    command_interfaces:
      - position 
    state_interfaces:
      - position
      - velocity

joints4_controllers:
  ros__parameters:
    joints:
      - joint4
    interface_name: position
    command_interfaces:
      - position 
    state_interfaces:
      - position
      - velocity 

joints5_controllers:
  ros__parameters:
    joints:
      - joint5
    interface_name: position
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity

joints6_controllers:
  ros__parameters:
    joints:
      - joint6
    interface_name: position
    command_interfaces:
      - position 
    state_interfaces:
      - position
      - velocity 

     
